### PR TITLE
[GST-2195] Travis change to include Ansible 2.1.1.0

### PR DESCRIPTION
**Pull request by quater**
Let's see what breaks so I can implement fixes.

### CODE REVIEW

**Comment by quater** [comment]
The Travis test was successful for Ansible 2.1.1.0 and thus no additional code changes are required.

File: tests/vagrant/test.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@
         - build
 
   roles:
-    - name: ansible-city.nginx
+    - role: ansible-city.nginx
```
Comment (lobsterdore) [inline tests/vagrant/test.yml]: Could you change ansible-city to sansible? ansible-city is an old version of this org, there will probably be other references in the makefile as well.

Comment (quater) [reply tests/vagrant/test.yml]: I have swapped `ansible-city` with `sansible` and squashed all commits into one. I believe it should be ready for merging.

**Comment by lobsterdore** [comment]
LGTM :+1:

**Pull request merged**